docs/mobile-apps/features.md

```diff
@@ -132,21 +125,41 @@ Try running one of the below example scripts, which demonstrate the default capa
   ```
   </details>
 
-<br/>
 
-#### **iOS Real Devices**
+#### Real Devices
+
+To enable Touch ID and Face ID on iOS and Android real devices, add the `allowTouchIdEnroll` capability in your test script and set it to `true`.
+
+When your test triggers a biometric prompt event, you can trigger a successful or failed authentication using the following commands:
 
-To enable Touch ID and Face ID on iOS real devices, add the `allowTouchIdEnroll` capability in your test script and set it to true, as shown in the example below:
```

Comment:
Do we not need to mention the "allowTouchIdEnroll" here anymore?

Reply:
It is my understanding from the sample code that you need both -- you have to set the `allowTouchIdCEnroll` capability in order to be able to execute the commands. @wswebcreation -- perhaps you can provide some insight on the exact requirements and whether they are different for iOS vs. Android. Thanks!